bionic/protocols.py

```diff
@@ -368,6 +369,32 @@ def write(self, df, path):
         dd.to_parquet(df, path, write_index=True)
 
 
+class YamlProtocol(BaseProtocol):
+    """
+    Decorator indicating that an entity's values can be serialized using the
+    ``PyYAML`` library.
+
+    Parameters
+    ----------
+    kwargs: keyword args for ``yaml.dump``
```

Comment:
super nit: We should probably write `` `**kwargs` `` instead of `kwargs` since these are keyword arguments.